telegram/chat.py

```diff
@@ -600,6 +601,10 @@ def promote_member(
 
         .. versionadded:: 13.2
 
+        ..versionchanged:: 13.12
+            Since Bot API 6.0, voice chat was renamed to video chat. The old parameter is
+            supported but is supplied as video_chats.
```

Comment:
what does `is supplied as video_chats` mean? I don't understand it without reading the code

Reply:
https://github.com/python-telegram-bot/python-telegram-bot/pull/3027#discussion_r871313141

Harshil suggested to drop that part completely in a different file, can do here as well

Reply:
also fine with me 👍